sway/tree/container.c

```diff
@@ -797,16 +797,16 @@ void container_set_floating(struct sway_container *container, bool enable) {
 			root_scratchpad_remove_container(container);
 		}
 		container_detach(container);
-		struct sway_container *reference =
+		struct sway_container *other =
 			seat_get_focus_inactive_tiling(seat, workspace);
-		if (reference) {
-			container_add_sibling(reference, container, 1);
-			container->width = reference->width;
-			container->height = reference->height;
+		if (other) {
+			container_add_sibling(other, container, 1);
+			container->width = other->width;
+			container->height = other->height;
 		} else {
-			workspace_add_tiling(workspace, container);
-			container->width = workspace->width;
-			container->height = workspace->height;
+			other = workspace_add_tiling(workspace, container);
```

Comment:
Let's just declare a new variable for this here rather than taking over `reference`/`other` from above to mean a different thing depending on what branch you're in.